lingetic-spring-backend/Dockerfile

```
@@ -2,23 +1,24 @@
 COPY build.gradle.kts .
 COPY settings.gradle.kts .
 
-COPY config/application.properties config/application.properties
+RUN --mount=type=secret,id=SENTRY_AUTH_TOKEN \
+    export SENTRY_AUTH_TOKEN=$(cat /run/secrets/SENTRY_AUTH_TOKEN) && \
+    ./gradlew dependencies --no-daemon
 
+COPY config/application.properties config/application.properties
 COPY src/main src/main
 
 RUN --mount=type=secret,id=SENTRY_AUTH_TOKEN \
```

Hadolint (2.12.0)

[warning] Quote this to prevent word splitting.

(SC2046)